src/error.rs

```diff
@@ -1,10 +1,9 @@
-use std::error::Error as StdError;
 use std::fmt;
 
 use modifier::Modifier;
 use {Response};
 
-pub use err::Error;
+pub use std::error::Error;
```

Comment:
We may want to cut this reexport. I only had it there as a (failed) attempt to keep the change roughly backwards compatible.